### PR TITLE
Added logos and icons

### PR DESCRIPTION
**Pull request by jordiferrero**
### Description of the change
Following #5, I added logo files in `docs/_static`.
Added logo to README file.

### Progress of the PR
- [x] Added files
- [x] ready for review.

### CODE REVIEW

**Comment by jordiferrero** [comment]
If anyone knows how to update the:
- Logo in ReadtheDocs
- Icon in ReadtheDocs
Please let me know to get that sorted too.

**Comment by ericpre** [comment]
It should be in https://github.com/hyperspy/rosettasciio/blob/c85d091cd7990898b17e175c6b05ccf5817e0693/docs/conf.py#L63-L89

and the corresponding documentation in:
https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/branding.html

**Comment by jlaehne** [comment]
> If anyone knows how to update the:
> 
>     * Logo in ReadtheDocs
> 
>     * Icon in ReadtheDocs
>       Please let me know to get that sorted too.

We use the pydata-sphinx-theme, so it should be according to this page:
https://pydata-sphinx-theme.readthedocs.io/en/v0.7.2/user_guide/configuring.html

For an implementation also handling the dark mode, see the documentation of pydata-sphinx-theme itself:
https://github.com/pydata/pydata-sphinx-theme/blob/main/docs/conf.py

**Comment by jordiferrero** [comment]
Thanks. I believe it is now ready for review.

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 # RosettaSciIO
 
+<img src="https://github.com/jordiferrero/rosettasciio/tree/main/docs/_static/logo_rec_oct22.svg" width="300" alt="RosettaSciIO">
```
Comment (francisco-dlp) [inline README.md:14]: Maybe this should point to https://github.com/hyperspy/rosettasciio ?

Comment (jordiferrero) [reply README.md]: Well spotted :-)

**Comment by jlaehne** [comment]
Splendid, just a small lint error now on `conf.py`.

**Review by ericpre — APPROVED** [review]

**Pull request merged**